README.md

```diff
@@ -6,9 +6,22 @@
 
 # Ruplacer
 
-Find and replace text in source files.
+Find and replace text in source files:
+
+```
+$ ruplacer old new src/
+Patching src/a_dir/sub/foo.txt
+-- old is everywhere, old is old
+++ new is everywhere, new is new
+
+Patching src/top.txt
+-- old is nice
+++ new is nice
+
+Would perform 2 replacements on 2 matching files.
+Re-run ruplacer with --go to write these changes to disk
+```
 
-![ruplacer screenshot](https://dmerej.info/blog/pics/ruplacer.png)
```

Comment:
The screenshot was nice, it had more colors and underlines than the simple snippet :(

Reply:
was disappointed but the non-uglyness also !